public/views/content/studentData.html

```diff
@@ -4,7 +4,7 @@
     <!-- Reading level -->
     <center><b>{{thisMonth | date: 'MMMM'}}</b></center>
     <br>
-    <span tooltip-placement="left" uib-tooltip="Current reading level" class="glyphicon glyphicon-book" aria-hidden="true"></span> Lvl X
+    <span tooltip-placement="left" uib-tooltip="Current reading level" class="glyphicon glyphicon-book" aria-hidden="true"></span> Lvl {{currentStudent.reading}}
```

Comment:
the reason i took it out is because i thought we weren't having reading level be apart of the studentmodel anymore. are we going to do the "make latest reading level entry be put into student model" solution?